server/src/metadata.rs

```diff
@@ -139,14 +139,12 @@ impl STREAM_INFO {
                 .and_then(parse_string)
                 .map_err(|_| Error::SchemaNotInStore(stream.name.to_owned()));
 
-            let metadata = match (alert_config, schema) {
-                (Ok(alert_config), Ok(schema)) => LogStreamMetadata {
-                    schema,
-                    alert_config,
-                    ..Default::default()
-                },
-                _ => continue,
+            let metadata = LogStreamMetadata {
+                schema: schema.unwrap_or_default(),
+                alert_config: alert_config.unwrap_or_default(),
```

Comment:
Instead of returning or not adding metadata to stream we add empty string for now. This allows first event to not fail in case parseable was restarted just after adding a stream